homeassistant/components/plex/server.py

```diff
@@ -669,6 +669,120 @@ def lookup_media(self, media_type, **kwargs):
             _LOGGER.error("%s not found in %s", failed_item, library_name)
             return None
 
+    def continue_media(self, media, library_name, continuation):
+        """Find the episode object from which to resume watching.
+
+        The following continuation modes are supported:
+        1. deck
```

Comment:
To keep the terminology consistent.
```suggestion
        1. on_deck
```

Reply:
This suggestion seemed totally reasonable at first, but this caused difficulties because mode arguments are delimited by '_'.

For example 'unwatched_first' is mode: 'unwatched' and arg: 'first'; However, 'on_deck' will parse into mode 'on' and arg 'deck' which would require special handling. I ended up changing this to "ondeck" instead, to try and find a compromise - but I'm happy to add special handling if you strongly prefer "on_deck" (I've no strong feelings either way).